### PR TITLE
Rename change log message for uncategorized issues

### PR DESCRIPTION
**Pull request by raminqaf**

### CODE REVIEW

**Comment by irux** [comment]
Fix or rename ?

**Comment by disrupted** [comment]
> Fix or rename ?

looks like rename to me

**Review by disrupted — APPROVED** [review]
LGTM, just set the title of the PR to rename instead of fix

File: .github/changelog-config.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     ],
     "ignore_labels": ["ignore"],
     "sort": {"order": "ASC", "on_property": "mergedAt"},
-    "template": "# [${{TO_TAG}}](https://github.com/bakdata/kpops/releases/tag/${{TO_TAG}}) - Release Date: ${{TO_TAG_DATE}}\n\n${{CHANGELOG}}\n\n<summary>Uncategorized</summary>\n\n${{UNCATEGORIZED}}",
+    "template": "# [${{TO_TAG}}](https://github.com/bakdata/kpops/releases/tag/${{TO_TAG}}) - Release Date: ${{TO_TAG_DATE}}\n\n${{CHANGELOG}}\n\n## 🌀 Miscellaneous\n\n${{UNCATEGORIZED}}",
```
Comment (sujuka99) [inline .github/changelog-config.json:10]: Shouldn't it be like this?
```suggestion
    "template": "# [${{TO_TAG}}](https://github.com/bakdata/kpops/releases/tag/${{TO_TAG}}) - Release Date: ${{TO_TAG_DATE}}\n\n${{CHANGELOG}}\n\n### 🌀 Miscellaneous\n\n${{UNCATEGORIZED}}",
```

**Review by sujuka99 — DISMISSED** [review]

File: .github/changelog-config.json (hunk 1 above)
Comment (raminqaf) [reply .github/changelog-config.json:10]: Nope, the titles are all header 2 (##)

**Review by irux — DISMISSED** [review]
LGTM!

**Review by irux — APPROVED** [review]
LGTM

**Review by sujuka99 — APPROVED** [review]

**Pull request merged**